docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -160,10 +162,39 @@ The following configuration properties are available:
        This flag allows ignoring the value provided in the metadata document.
        Default is ``true``.
 
+.. _trino-oauth2-refresh-tokens:
+
 Refresh tokens
 ^^^^^^^^^^^^^^
 
-In order to start using refresh tokens flows, the following property must be
+*Refresh tokens* allow you to securely control the length of user sessions
+within applications. The refresh token has a longer lifespan (ttl) and is used
+to refresh the *access token* that has a shorter lifespan. With refresh tokens
+behind the scenes, users can stay logged in for a longer session without
```

Comment:
> users can stay logged in for a longer session without  (feels awkward)
users can extend their session without ...
or 
users can remain logged-in for an extended duration ...

Reply:
Thanks for your review comment Ben. It isn't really users extending their session, it is the refresh token extending the session. 

Will rework.

Reply:
@bendemott  please review again when you get a chance. Thank you.

Reply:
"without interruption by another login request" sounds like another login request is the means by which one works without interruption.

it's actually one token that allows another to refresh as per the second sentence.

How about: "The refreshed tokens prevent sessions from being interrupted by repeated log in requests."

Reply:
@m57lyra  "The refreshed tokens" plural sounds odd to me. The refresh token keeps fetching a new access token.